### PR TITLE
rewrite abstract

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

**Review by suhasHere — APPROVED** [review]

File: draft-lcurley-moq-transport.md [hunk 1]
```diff
@@ -52,9 +52,15 @@ informative:
...
-This document defines the core behavior for MoQTransport, a live media transport protocol over QUIC.
-The application fragments a live stream into objects, including a header that describes the basic relationship between objects.
-Objects are starved/dropped during congestion based on priorities in order to minimize latency.
+This document defines the core behavior for MoQTransport, a media
+transport protocol over QUIC.  MoQTransport allows a producer of media
+to publish data and have it consumed via subscription by a multiplicity
+of endpoints. It supports intermediate content distribution networks and
+is designed for high scale and low latency distribution. The core
+subscribable entities are tracks, consisting of a sequence of objects
+organized into groups. MoQTransport is a generic protocol, designed to
```
Comment (kixelated) [inline draft-lcurley-moq-transport.md:61]: > The core subscribable entities are tracks, consisting of a sequence of objects organized into groups.

I wouldn't introduce terminology in the abstract. This sentence doesn't add anything or make me think "boy I want to read more".

**Review by kixelated — COMMENTED** [review]
The new text is super vague but that's fine for an abstract.

File: draft-lcurley-moq-transport.md (hunk 1 above)
Comment (afrind) [reply draft-lcurley-moq-transport.md:61]: @kixelated do you need that line removed now or can you stamp?

Comment (suhasHere) [reply draft-lcurley-moq-transport.md:61]: I think Luke has a fair point. We should make that change.

**Review by kixelated — DISMISSED** [review]
Could you revert your changes Suhas? The previous commit was nice, I just wanted a sentence removed.

**Comment by suhasHere** [comment]
@kixelated  done and also removed the last sentence

**Review by kixelated — APPROVED** [review]

**Pull request merged**